docs/src/configuration/app/build.md

```diff
@@ -124,6 +130,8 @@ What is "safe" to run in a `post_deploy` hook vs. in a `deploy` hook will vary b
 
 The `post_deploy` hook logs to its own file in addition to the activity log, `/var/log/post-deploy.log`.
 
+The `post_deploy` hook is the only hook provided that will run from the beginning during a redeploy. 
```

Comment:
@chadwcarlson @MariadeAnton 

> from the beginning

I guess you're saying it runs "every time"? I'm not sure what the beginning refers to

Reply:
... googled.. maybe that's a false friend of _d'emblée_? :fr: